src/libraries/System.Private.CoreLib/src/System/Text/Ascii.Utility.cs

```diff
@@ -1394,6 +1394,8 @@ private static bool VectorContainsNonAsciiChar(Vector128<byte> asciiVector)
             }
         }
 
+
+
```

Comment:
for some reason I can not provide a single suggestion to remove both of the empty lines

```suggestion
```

Reply:
I've removed the whitespace and the unused helpers (inluding a 3rd one with no references)